### PR TITLE
add session information component

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[UI sollte Session-token anzeigen](https://trello.com/c/xxVVI1rZ/1214-ui-sollte-session-token-anzeigen)

File: projects/core/src/lib/users/user-session/user-session.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    ngOnDestroy(): void {
+        this.sessionStreamSubscription.unsubscribe();
+    }
+
+    copySessionTokenToClipboard(): void {
+        if (this.session) {
+            this.clipboard.copy(this.session.sessionToken);
+        }
```
Comment (ChristianBeilschmidt) [inline projects/core/src/lib/users/user-session/user-session.component.ts:31]: Somehow, the copy button isn't working for me.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/core/src/lib/users/user-session/user-session.component.ts (hunk 1 above)
Comment (jdroenner) [reply projects/core/src/lib/users/user-session/user-session.component.ts]: [643615e](https://github.com/geo-engine/geoengine-ui/pull/585/commits/643615e44353906280602288cef60eceb5f0f443)

File: projects/core/src/lib/users/user-session/user-session.component.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ng-container>
+    <dl>
+        <dt>Session token</dt>
+        <dd>{{ session?.sessionToken }} <button mat-stroked-button>copy</button></dd>
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply projects/core/src/lib/users/user-session/user-session.component.html]: [643615e](https://github.com/geo-engine/geoengine-ui/pull/585/commits/643615e44353906280602288cef60eceb5f0f443)

File: projects/core/src/lib/users/user-session/user-session.component.scss [hunk 3]
```diff
@@ -1,9 +1,32 @@
-dl {
-    margin: 0;
-    dt {
+.pair {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start;
```
Comment (ChristianBeilschmidt) [inline projects/core/src/lib/users/user-session/user-session.component.scss:5]: From the compiler:
> WARNING: autoprefixer: /home/beilschmidt/git/geoengine-ui/projects/core/src/lib/users/user-session/user-session.component.scss:5:3: start value has mixed support, consider using flex-start instead

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/core/src/lib/users/user-session/user-session.component.scss (hunk 3 above)
Comment (jdroenner) [reply projects/core/src/lib/users/user-session/user-session.component.scss]: [caf80ff](https://github.com/geo-engine/geoengine-ui/pull/585/commits/caf80ff70acc4e70cccb1ceffa616ed062bfe61e)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**